sections/futures/UserInfo/PositionsTable.tsx

```diff
@@ -220,7 +220,10 @@ const PositionsTable: FC<FuturesPositionTableProps> = () => {
 						<PositionCell>
 							<FlexDivRowCentered columnGap="5px">
 								<PnlContainer>
-									<Currency.Price price={row.position.pnl} colored />
+									<Currency.Price
+										price={row.position.pnl.add(row.position.accruedFunding)}
```

Comment:
We didn't count funding in unrealized pnl since it is applied directly to your margin, so it's technically realized immediately. Based on what I'm seeing, if you interacted with this position the uP&L would be reset to zero, realized pnl would be on the "Trades" tab, and the funding value here would continue to accrue.

This change only applied `accruedFunding` which is like the funding since the last interaction. There is an additional `netFunding` value on the position that accounts for the other portion.

Reply:
ah should use netFunding then

Reply:
I'm suggesting it shouldn't be considered unrealized pnl at all. It is realized, because it is added or deducted from your margin immediately.

Reply:
pretty sure we realize pnl as well(in margin) whenever we make any changes to the position, but we still show it in UP&L

Reply:
Yeah, funding is realized, but funding can also move in both direction and already paid/deducted funding can be earnt back, same as realized losses can be made back before closing the position?

Reply:
I don't know what to say since I've made my position very clear. You're correct that funding could flip.

Since funding has both a part that's "already accred" and "accruing" you could possibly consider the "accruing" part as unrealized, but it's very confusing for users. This means they accrue funding (realize pnl) when they interact with the position, which includes things like order submission, margin transfers, and order confirmations. It's difficult to explain to a user why they realized some profit or loss when they make a margin transfer.